### PR TITLE
Updated layout after some months of testing.

### PR DESCRIPTION
**Pull request by finex**
The current setup looks quite better than the initial one. It's not
perfect but it allows me to work quite well.

The only big problem is the Alt modifiers on the home row which
sometimes break the typing if keys are pressed too fast. I'm also not
satisfied by the TAB key position on the thumb cluster.

Main improvements since previous push:
- Shift layouts on both halves
- Moved around symbols
- Improved number/fn keys
- Reorganized thumbs

### CODE REVIEW

File: keyboards/redox/keymaps/finex/keymap.c [hunk 1]
```diff
@@ -3,47 +3,159 @@
 extern keymap_config_t keymap_config;
 
 // Layer names
-#define _QWERTY 0
-#define _COLEMAKDHMK 1
-#define _QWERTY_MAC 2
-#define _COLEMAKDHMK_MAC 3
+// - Base layers:
+#define _COLEMAKDHMK 0
```
Comment (drashna) [inline keyboards/redox/keymaps/finex/keymap.c]: You may want to consider using an `enum` here, as it may be easier to manage in the future.

Comment (finex) [reply keyboards/redox/keymaps/finex/keymap.c]: Thank you @drashna for the suggestions. I've used an enum and committed to my master branch (next time I will create a branch). Tell me if it is ok :-)

Comment (drashna) [reply keyboards/redox/keymaps/finex/keymap.c]: Yup, that's great

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**